sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -251,6 +251,19 @@ internal static RecognizeHealthcareEntitiesResultCollection ConvertToRecognizeHe
 
         #endregion
 
+        #region Analyze Operation
+
+        internal static AnalyzeOperationResult ConvertToAnalyzeOperationResult(AnalyzeJobState jobState, IDictionary<string, int> map)
```

Comment:
what about all the properties that are part of `JobMetadata`?

Reply:
We are keeping the important properties in AnalyzeOperationResult, if required we can add them here later. Is there any specific property you are looking for?

Reply:
well I don't know the use of all properties so I can't tell which ones are important. But if the service returns them we usually expose them unless we have a reason not to.
Because this is codegenerator with more reason I would have thought that all properties are listed.

Now, you don't need to and I am ok if they are not exposed in this preview, but also, curios about the other languages and the exposure

Reply:
@mssfang what are you doing in Java?

Reply:
In java, I transform the `AnalyzeJobState` to the explored model `AnalyzeTasksResult`.
https://github.com/Azure/azure-sdk-for-java/pull/17267/files?file-filters%5B%5D=.java&file-filters%5B%5D=.json#diff-8e22c3893953fdc22077905634905ba868d6193eddba53bffb4354e6fdf28886R297

Reply:
@suhas92 This is not a blocker for this PR. You can add this properties in a future PR.